src/Features/Core/Portable/EditAndContinue/Deltas.cs

```diff
@@ -30,12 +59,28 @@ internal sealed class Deltas
             ImmutableArray<(Guid ThreadId, ActiveInstructionId OldInstructionId, LinePositionSpan NewSpan)> activeStatementsInUpdatedMethods)
         {
             Mvid = mvid;
-            IL = new ILDelta(il);
-            Metadata = new MetadataDelta(metadata);
-            Pdb = new PdbDelta(pdb, updatedMethods);
+            IL = il;
+            Metadata = metadata;
+            Pdb = pdb;
+            UpdatedMethods = updatedMethods;
             NonRemappableRegions = nonRemappableRegions;
             ActiveStatementsInUpdatedMethods = activeStatementsInUpdatedMethods;
             LineEdits = lineEdits;
         }
+
+        public Data Serialize()
```

Comment:
nit: we've generally gone with Hydrate/Dehydrate. Serialize/Deserialize is usually teh terms we use when explicitly reading/writing from a byte-stream abstraction.  But if you prefer these terms, that's fine.